parts/3-clinical/slides.md

```diff
@@ -0,0 +1,139 @@
+# Outline, slides
+
+
+## Clinical Genomics
+Core facility at SciLifeLab
+
+### How are we different from NGI?
+Bridge the gap between sending raw data to computer savvy researchers vs.
+only somewhat computer litterate genetecists and doctors. Basically it
+means processing the data as far as possible and intuitively visualize it.
+
+### Mission
+5 min analysis
+
+
+## Genomics in healthcare today
+In fact, clinics already routinely send in patient samples for sequencing.
+This is pretty cool because healthcare is *not* famous for moving fast or
+adopting new technology.
+
+The focus area for us is diagnosis of rare inherited disoders. They are
+an ideal case to bring genomics into the clinic. The symptoms are often
+severe and develop early in life, the cause is 100% genetic, and the
+cause-effect relationship is therefore rather simple. Basically, you
+find a defect gene and there's your cause. But figuring out what gene you
+are looking for isn't quite so easy.
+
+If we step back a bit; these patients can go undiagnosed for 15 years,
+if they survive that long. Each individual test can only ever give
+indications of what the cause of the disorder really is.
+
+In contrast, using DNA sequencing technologies, we are able to go straight
+for the root cause of the disorder and deliver conclusive results in *two
+weeks*.
+
+So why are people still hesitant?
+
+1. The high initial cost. One challenge is to prove that sequencing can
+   effetively replace many of the old inconclusive tests.
+
+2. Analysis of data is neither trivial nor standardized. For example how
+   do you distingish an error from a mutation? Essentially 99.9% of the
+   data generated is either too difficult to interpret or completely
+   non-informative. Most sequences are just the same as what we know we
+   find healthy individuals. To be honest we are generating a lot of
+   "waste" that no ones has figured out a plan to do with. Kind of like
+   a nuclear power plant.
+
+As the patient sample travels through our facility thousands of lines of
+Python code will eventually touch it. I'm going to focus on the delivery
+report.
+
+
+## Excel sheet -> Web interface
+What these guys [point to Guillermo] are delivering to researchers is very
```

Comment:
hmm that's not completely true, we do the analysis and what we give back are sample and project reports (and raw data). It is true though that as far as I know most of the time researchers just re-run our analysis :-/ but they don't have to